src/commands/init/index.ts

```diff
@@ -0,0 +1,93 @@
+import prompts from 'prompts';
+import { checkIfConfigExists, createConfigFile } from '../../config-file';
+import * as output from '../../output';
+
+export async function init() {
+  try {
+    await initInternal();
+  } catch (error) {
+    output.clearLine();
+    output.outputError(error);
+    process.exit(1);
+  }
+}
+
+async function initInternal() {
+  const configExists = checkIfConfigExists();
+
+  if (configExists) {
+    const response = await prompts({
+      type: 'confirm',
+      message: 'Config found, do you want to re-initialize it?',
+      name: 'reinitialize',
+    });
+
+    if (!response.reinitialize) {
+      output.outputBold('Cancelling initialization');
+      return;
+    }
+  }
+
+  output.outputBold("Welcome to AI CLI. Let's set you up quickly.");
+
+  const response = await prompts([
+    {
+      type: 'select',
+      name: 'provider',
+      message: 'Which inference provider would you like to set up:',
+      choices: [
+        { title: 'OpenAI', value: 'openAi' },
+        { title: 'Perplexity', value: 'perplexity' },
+      ],
+      initial: 0,
+      hint: '',
+    },
+    {
+      type: 'confirm',
+      message: 'Do you already have OpenAI API key?',
```

Comment:
You are asking about OpenAI api key, what about case when user wanted to provide only Perplexity key?